app/components/avo/actions_component.rb

```diff
@@ -86,8 +86,9 @@ def icon(action)
   end
 
   def render_item(action)
-    if action.is_a?(Avo::DividerComponent)
-      render Avo::DividerComponent.new
+    if action.is_a?(Avo::Divider)
+      label = action.label.is_a?(Hash) ? action.label[:label] : nil
+      render Avo::DividerComponent.new(label)
     else
       render_action_link(action)
```

Comment:
Let's use a `case` statement and instead of `else`, check for an actual action class.

Reply:
Ok